river/metrics/balanced_accuracy.py

```diff
@@ -0,0 +1,56 @@
+from . import base
+from . import confusion
+
+__all__ = ['BalancedAccuracy']
+
+class BalancedAccuracy(base.BinaryMetric, base.MultiClassMetric):
```

Comment:
No need for inheriting from `base.BinaryMetric`, it is implied by `base.MultiClassMetric`.